.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,143 @@
+name: 🐞 Bug Report
+description: File a bug report
+labels: ["bug", "needs-triage"]
+# assignees:
+#   - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there 👋,
+
+        Thank you for reporting the bug!
+
+        In case you want a new resource, data source or a new attribute supported in one of those,
+        please request it as a feature!
+  - type: input
+    id: provider-version
+    attributes:
+      label: GitLab Provider version
+      description: What version of the `gitlabhq/gitlab` provider are you using?
+      placeholder: "3.9.1"
+    validations:
+      required: true
+  - type: input
+    id: gitlab-version
+    attributes:
+      label: GitLab version
+      description: What version of GitLab are you using?
+      placeholder: "14.8"
+    validations:
+      required: true
+  - type: input
+    id: terraform-version
+    attributes:
+      label: Terraform version
+      description: What version of terraform are you using?
+      placeholder: "Please paste the output of `terraform -v`` here"
+    validations:
+      required: true
+  - type: dropdown
+    id: affected-resources
+    attributes:
+      label: Affected Resources / Data Sources
+      description: What resources and / or data sources are affected?
+      multiple: true
+      options:
+        - other
+        - datasource/gitlab_group_membership
```

Comment:
@armsnyder would you be ok if I generate this list (and therefore this file) during `make reviewable` depending on the source code?

Reply:
Yes, that's actually right, I'll remove it entirely.